### PR TITLE
Aria fixes

### LINKED ISSUE #167: Errors and warning whenever running download service
**Describe the bug**

Running download service there are some info stating there is `[WARN]` or  `[ERROR]` info.

**Which UI**
none :)

**Hardware / Software**
Doesn't matter

**Steps to Reproduce**
1. `docker compose --profile download up --build`
2. Boom...
```
Attaching to webui-docker-download-1
webui-docker-download-1  | Downloading, this might take a while...
webui-docker-download-1  | 
webui-docker-download-1  | **10/22 20:46:12 [WARN] Unknown option:   # this is the only way aria2c won't fail if the file already exists**
webui-docker-download-1  | 
webui-docker-download-1  | 10/22 20:46:12 [NOTICE] Downloading 7 item(s)
webui-docker-download-1  | 
webui-docker-download-1  | **10/22 20:46:12 [ERROR] Exception caught while loading DHT routing table from /root/.cache/aria2/dht.dat**
webui-docker-download-1  | **Exception: [DHTRoutingTableDeserializer.cc:82] errorCode=1 Failed to load DHT routing table from /root/.cache/aria2/dht.dat**
webui-docker-download-1  | 
webui-docker-download-1  | 10/22 20:46:12 [NOTICE] IPv4 DHT: listening on UDP port 6958
webui-docker-download-1  | 
webui-docker-download-1  | 10/22 20:46:12 [NOTICE] IPv4 BitTorrent: listening on TCP port 6943
webui-docker-download-1  | 
webui-docker-download-1  | **10/22 20:46:12 [ERROR] IPv6 BitTorrent: failed to bind TCP port 6943**
webui-docker-download-1  | Exception: [SocketCore.cc:312] errorCode=1 Failed to bind a socket, cause: Name does not resolve
```
Despite all those scary warnings and errors it still manages to pull it off and download everything, so I would treat is as low priority.
First `[WARN]` could be corrected by not inserting comment there (aria2c manual states that line is a comment if it starts with `#`, but it seems that spaces before `#` sign are not ignored.
ipv6 could probably be omitted with `--disable-ipv6` option for aria2c. This error may be system specific as I've got ipv6 disabled on my box (if so, there should be method to verify ipv6 connectivity first to eventually provide proper options for aria2c).

### PR DESCRIPTION
**Pull request by DevilaN**
Fixes WARN / ERRORs addressed in https://github.com/AbdBarho/stable-diffusion-webui-docker/issues/167
Closes #167

### CODE REVIEW

File: services/download/download.sh [hunk 1]
```diff
@@ -4,9 +4,14 @@ set -Eeuo pipefail
 
 mkdir -p /data/.cache /data/StableDiffusion /data/Codeformer /data/GFPGAN /data/ESRGAN /data/BSRGAN /data/RealESRGAN /data/SwinIR /data/LDSR /data/ScuNET /data/embeddings
 
+ARIA2C_OPTS=""
+if [ "`ip -6 addr`" == "" ]; then
+    ARIA2C_OPTS="${ARIA2C_OPTS} --disable-ipv6"
+fi
+
```
Comment (AbdBarho) [inline services/download/download.sh:11]: is this really necessary? just disable ipv6 totally, also, is it possible to have this flag only on the v1.5 model in the `links.txt`? maybe `disable-ipv6=true`?

**Review by AbdBarho — CHANGES_REQUESTED** [review]
Thanks! sorry for being late for other stuff.

File: services/download/download.sh (hunk 1 above)
Comment (DevilaN) [reply services/download/download.sh:11]: Checking IPV6 makes me sleep better :-)
In contrast to https://github.com/moby/moby/issues/32850 which seems to wait over 5 years, and nobody knows if tomorrow there will be another user presenting some edge case of docker use on theirs system ipv6 exclusive.
OK. I've explained myself. Now... Your solution with flag seems to be better. I am sorry - I do not have even basic knowledge about aria2c. I will look onto it in manual and probably will commit change without unnecessary code.

Best regards

Comment (AbdBarho) [reply services/download/download.sh:11]: ayy man no worries man, I don't know `aria2c` either :')

I just googled `cli download utility linux` and it was third after `curl` and `wget`, both of them failed on my extremely slow and unstable internet connection.

Comment (DevilaN) [reply services/download/download.sh]: > is this really necessary? just disable ipv6 totally, also, is it possible to have this flag only on the v1.5 model in the `links.txt`? maybe `disable-ipv6=true`?

There is no ipv6 specific option per link ( https://aria2.github.io/manual/en/html/aria2c.html#input-file )
After some research yesterday I think `--disable-ipv6` option for entire command is only way to go.
I've commited changes so merge is now possible.

**Review by AbdBarho — APPROVED** [review]

**Pull request merged**